src/ds/d400/d400-private.h

```diff
@@ -39,6 +39,7 @@ namespace librealsense
         const uint16_t RS455_PID = 0x0B5C; // D455
         const uint16_t RS457_PID = 0xabcd; // D457
         const uint16_t RS457_RECOVERY_PID = 0xbbcd; // D457 DFU Recovery
+        const uint16_t RS430_GMSL_PID = 0xabce; // D450 GMSL
```

Comment:
If we do reuse the same recovery PID as D457, let's rename it
Currently it's `RS457_RECOVERY_PID`

Reply:
will look into if it's necessary to have separate recovery pid.

Reply:
Updated to RS400_MIPI_RECOVERY_PID